.github/workflows/build-frontend.yaml

```diff
@@ -0,0 +1,27 @@
+name: Build frontend
+
+on:
+  pull_request:
+      branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
```

Comment:
Add cache to npm deps https://github.com/actions/cache/blob/master/examples.md#node---npm